src/components/SwipeButton/index.js

```diff
@@ -94,6 +94,7 @@ const SwipeButton = props => {
     thumbIconImageSource,
     thumbIconStyles,
     title,
+    titleMaxFontScale,
```

Comment:
Hi @ah584d , thanks for PR. Can you please keep the prop in sorted order?

Reply:
fixed